guaranteed-message-delivery/README.md

```diff
@@ -0,0 +1,389 @@
+            ReservationResponse|http:ClientError reservationResponse = 
+                hospitalBackend->/[content.hospital_id]/categories/[content.category]/reserve.post({
+                    patient: content.patient,
+                    doctor,
+                    hospital,
+                    appointment_date: content.appointment_date
+                });
+
+            string smsBody;
+            if reservationResponse is http:ClientError {
+                log:printError("Reservation request failed", patient = patientName,
+                                                            doctor = doctor,
+                                                            hospital = hospital);
+                smsBody = string `Dear ${patientName
+                            }, your appointment request at ${hospital
+                            } failed. Please try again.`;
+            } else {
+                smsBody = string `Dear ${patientName
+                            }, your appointment has been accepted at ${hospital
+                            }. Appointment No: ${reservationResponse.appointmentNumber}`;
+            }
+
+            twilio:SmsResponse|error smsApiStatus = twilioEp->sendSms(fromNumber, content.patient.phone, smsBody);
+
+            if smsApiStatus !is twilio:SmsResponse {
+                log:printError("Failed to send an SMS message", smsApiStatus, phoneNo = content.patient.phone);
+            }
+        }
+    }
+    ```
+
+    - Extract the necessary values to variables and send a `POST` request to the hospital service to reserve the appointment. The `hospital_id` and `category` values are used as path parameters.
+
+    - Use the `is` check to decide the flow based on the response to the client call. If the request failed with an error, log the error and set an error message to the `smsBody` variable, else, set a confirmation message.
+
+    - Call the twilio endpoint with the `smsBody` and, sender's and patient's phone numbers.
+
+#### Complete source
+
+<TODO: Complete source code>
+
+### Step 3: Build and run the service
+
+![Run the service](./resources/run_the_service.gif)
+
+> **Note:**
+> Alternatively, you can run this service by navigating to the project root and using the `bal run` command.
+>
+> ```
+> guaranteed-message-delivery$ bal run
+> Compiling source
+>         integration_tutorials/guaranteed-message-delivery:0.1.0
+>
+> Running executable
+> ```
+
+### Step 4: Try out the use case
+
+Let's test the use case by sending a request to the service.
+
+#### Start the backend service
+
+Download the JAR file for the [backend service](https://github.com/ballerina-guides/integration-tutorials/blob/main/backends/hospital-service/hospitalservice.jar) and execute the following command to start the service.
+
+```
+$ bal run hospitalservice.jar
+```
+
+#### Start the rabbitmq message broker
+
+Follow the [rabbitmq guidelines ](<link to the guidelines>) to configure and start the rabbitmq message broker.
```

Comment:
```suggestion
Follow the [rabbitmq guidelines](<link to the guidelines>) to configure and start the rabbitmq message broker.
```

rabbitmq -> RabbitMQ 

Please fix wherever applicable for all applicable terms.